docs/howtos/upgrade_guide.md

```diff
@@ -93,6 +99,10 @@ Before you start the upgrade, move your custom settings from
 Ubuntu) or from `/etc/percona-xtradb-cluster.conf.d/wsrep.cnf` (on Red Hat
 and CentOS) to the new location accordingly.
 
+!!! note
+
+    If you have moved your my.cnf file to a different location and added a symlink to `/etc/my.cnf`, the the RPM package manager, when upgrading, can delete the symlink and put a default my.cnf file in /etc/.
```

Comment:
double 'the' 
`the the RPM package manager`

Reply:
fixed.